### PR TITLE
Add Traefik Hub access and remove Pilot access

### PR DESCRIPTION
**Pull request by tomMoulard**
### What does this PR do?

The purpose of this PR is to integrate direct access to Traefik Hub, Traefik Labs' latest product, from the Traefik Proxy dashboard.

This PR also removes the direct access to Traefik Pilot. For Traefik Pilot users, Traefik Pilot continues to be available and is accessible at [pilot.traefik.io](http://pilot.traefik.io/).

If you're already part of the Traefik Hub Private Beta, you can now test Traefik Hub. For the others, please stay tuned.

### Motivation

PR #8837

### More

~- [ ] Added/updated tests~
- [X] Added/updated documentation

### CODE REVIEW

File: docs/content/plugins/index.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Plugins and Traefik Pilot
 
+!!! warning "Pilot deprecation"
+
+    Traefik Pilot is deprecated and will be removed soon. Please check our Blog for migration instructions later this year.
```
Comment (kevinpollet) [inline docs/content/plugins/index.md:5]:
```suggestion
    Traefik Pilot is deprecated and will be removed soon.
    Please check our Blog for migration instructions later this year.
```

File: docs/content/migration/v2.md [hunk 2]
```diff
@@ -452,3 +452,11 @@ the value for the method label becomes `EXTENSION_METHOD`, instead of the reques
...
 
 In `v2.6.1`, the Datadog tags added to a span changed from `service.name` to `traefik.service.name` and from `router.name` to `traefik.router.name`.
+
+## v2.7
+
+### Pilot
+
+`--pilot.token` and `--pilot.dashboard` are deprecated.
+Those fields will be removed in future 2.x releases.
+Please check our Blog for migration instructions later this year.
```
Comment (kevinpollet) [inline docs/content/migration/v2.md:462]:
```suggestion
In `v2.7`, the `pilot.token` and `pilot.dashboard` options are deprecated.
Please check our Blog for migration instructions later this year.
```

File: docs/content/migration/v2.md [hunk 3]
```diff
@@ -452,3 +452,11 @@ the value for the method label becomes `EXTENSION_METHOD`, instead of the reques
 ### Tracing
 
 In `v2.6.1`, the Datadog tags added to a span changed from `service.name` to `traefik.service.name` and from `router.name` to `traefik.router.name`.
+
+## v2.7
+
+### Pilot
```
Comment (kevinpollet) [inline docs/content/migration/v2.md:458]:
```suggestion
### Traefik Pilot
```

File: docs/content/plugins/index.md [hunk 4]
```diff
@@ -1,5 +1,10 @@
 # Plugins and Traefik Pilot
 
+!!! warning "Pilot deprecation"
```
Comment (kevinpollet) [inline docs/content/plugins/index.md:3]:
```suggestion
!!! warning "Traefik Pilot Deprecation"
```

**Review by kevinpollet — APPROVED** [review]
LGTM 👀

**Review by rtribotte — APPROVED** [review]
LGTM 👌

**Review by mpl — APPROVED** [review]

**Pull request merged**